githubutil/githubutil.go

```diff
@@ -13,10 +17,13 @@ import (
 	"strings"
 	"time"
 
+	"github.com/golang-jwt/jwt/v5"
 	"github.com/google/go-github/v65/github"
 	"golang.org/x/oauth2"
 )
 
+const githubAPI = "https://api.github.com"
```

Comment:
`githubAPI` isn't all that descriptive. This is also only used in one place:

```go
	url := fmt.Sprintf("%s/app/installations/%d/access_tokens", githubAPI, installationID)
```

Inlining it arguably makes it clearer what the URL will be in context:

```go
	url := fmt.Sprintf("https://api.github.com/app/installations/%d/access_tokens", installationID)
```